sdk/schemaregistry/perf-tests/schema-registry-avro/sample.env

```diff
@@ -0,0 +1,20 @@
+SCHEMA_REGISTRY_ENDPOINT=<Endpoint URL>
+SCHEMA_REGISTRY_GROUP=<Group name for schemas in registry>
+
```

Comment:
Doesn't look like all the secrets are being used?

Reply:
Good point.